### PR TITLE
Create gscroll.lic

### PR DESCRIPTION
**Pull request by Raykyn55**
like gbox, for scrolls.

### CODE REVIEW

File: gscroll.lic [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      [
+        { name: 'container', regex: /\w+/i, variable: true, description: 'Name of the container to get scrolls from' },
+        { name: 'player', regex: /\w+/i, variable: true, description: 'Name of the player to give scrolls to' }
+      ]
+    ]
+
+    args = parse_args(arg_definitions)
+    EquipmentManager.new.empty_hands
+
+    DRC.rummage('SC', args.container).each { |item| hand_over(args.container, item, args.player) }
```
Comment (MahtraDR) [inline gscroll.lic:15]:
```suggestion
    DRC.get_scroll_list_in_container(args.container).each { |item| hand_over(args.container, item, args.player) }
```

File: gscroll.lic [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  end
+
+  def hand_over(container, item, person)
+    return if DRCI.get_item?(item, container) && DRCI.give_item?(person, item)
```
Comment (MahtraDR) [inline gscroll.lic:19]:
```suggestion
  def hand_over(source_container, item, person)
    return if DRCI.get_item?(item, source_container) && DRCI.give_item?(person, item)
```

**Comment by MahtraDR** [comment]
Looks good. Made a couple small suggestions.

**Comment by MahtraDR** [comment]
Did not mean to close, misclick.

File: gscroll.lic [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      ]
+    ]
+
+    args = parse_args(arg_definitions)
+    EquipmentManager.new.empty_hands
+
+    DRCI.get_scroll_list_in_container(args.container).each { |scroll| hand_over(args.container, scroll, args.player) }
+  end
+
+  def hand_over(source_container, item, person)
```
Comment (Raykyn55) [inline gscroll.lic:18]:
```suggestion
  def hand_over(source_container, scroll, person)
```

File: gscroll.lic [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    return if DRCI.get_item?(item, source_container) && DRCI.give_item?(person, item)
```
Comment (Raykyn55) [inline gscroll.lic:19]:
```suggestion
    return if DRCI.get_item?(scroll, source_container) && DRCI.give_item?(person, scroll)
```

File: gscroll.lic [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+    DRCI.stow_item?(item)
```
Comment (Raykyn55) [inline gscroll.lic:21]:
```suggestion
    DRCI.stow_item?(scroll)
```

**Pull request merged**